101-backup-la-reporting/azuredeploy.json

```diff
@@ -0,0 +1,176 @@
+{
+    "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion":"1.0.0.0",
+    "parameters":{
+      "workspaceLocation": {
+         "type": "string",
+         "metadata": {
+           "description": "Specify the workspace region"
+         }
+       },
+      
+       "workspaceName":{
+          "type":"string",
+          "metadata":{
+             "description":"Specify the workspace name"
+          }
+       },
+       "_artifactsLocation":{
+          "type":"string",
+          "defaultValue":"https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-backup-la-reporting/",
+          "metadata":{
+             "description":"The base URI where artifacts required by this template are located"
+          }
+       },
+       "_artifactsLocationSasToken":{
+          "type":"securestring",
+          "defaultValue":"",
+          "metadata":{
+             "description":"The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated"
+          }
+       }
+    },
+    "variables":{
+       "nestedTemplates":{
+          "backup_jobs_non_log":"[concat(parameters('_artifactsLocation'), 'nestedtemplates/backup_jobs_non_log.json', parameters('_artifactsLocationSasToken'))]",
```

Comment:
use the uri function for the uris, e.g.

"backup_jobs_non_log":"[uri(parameters('_artifactsLocation'), concat('nestedtemplates/backup_jobs_non_log.json', parameters('_artifactsLocationSasToken')))]",

Reply:
Updated